coalib/output/ConsoleInteraction.py

```diff
@@ -599,12 +623,21 @@ def print_actions(console_printer, section, actions, failed_actions):
                             values for the action. If the user did
                             choose to do nothing, return (None, None).
     """
-    choice = choose_action(console_printer, actions)
-
-    if choice == 0:
-        return None, None
-
-    return get_action_info(section, actions[choice - 1], failed_actions)
+    if section.name == 'cli':
+        if (section.get('reprompt').value == ''
+                or ast.literal_eval(section.get('reprompt').value)):
```

Comment:
Hey, `coala_utils` has it's own function that returns for booleans from string, please use it over ast.
Check it out here: https://gitlab.com/coala/coala-utils/blob/master/coala_utils/string_processing/StringConverter.py#L41